README.md

```diff
@@ -377,11 +379,17 @@ To using YARN cluster,
 1. Point the location of spark-assembly.jar with `spark.yarn.jar` property.
 1. Add Hadoop Conf dir such as `/etc/hadoop/conf` to the classpath in the executable script `bin/spark-notebook`:
 ```
-declare -r script_conf_file="/etc/default/spark-notebook"
-
-declare -r app_classpath="/etc/hadoop/conf:$lib_dir/...
-
-addJava "-Duser.dir=$(cd "${app_home}/.."; pwd -P)"
+export HADOOP_CONF_DIR=/etc/hadoop/conf (or any other means of setting environment variables)
+```
+1. Prepare the application master environment by adding the variables to the customSparkConf. This is how you would add variables that are commonly added via spark.yarn.am.extraJavaOptions. E.g. in order to run on Hortonworks HDP you need to add a definition for the variable `hdp.version`:)
```

Comment:
what about backticks around spark.yarn.am.extraJavaOptions 
and bold Hortonworks HDP here too ?